src/interaction.service.ts

```diff
@@ -7,17 +7,17 @@ const endpoint = 'locations';
 
 export default (axios: AxiosInstance) => {
   return {
-    getInteractions:(locationId:string, params: InteractionParams) => {
+    getInteractions:(locationId:string, params?: InteractionParams) => {
```

Comment:
since this is a method inside Interactions, it could be renamed to getByLocationId, then when it's imported the sentence could be something like Interaction.getByLocationId instead of interactions.getInteractions